calico-policies/private-network-isolation/calico-v3/us-south/allow-private-service-endpoint.yaml

```diff
@@ -73,6 +73,8 @@ spec:
       - 166.9.17.37/32
       - 166.9.17.38/32
       - 166.9.17.39/32
+      - 166.9.22.216/32
```

Comment:
These aren't the right IPs, you want the ones from the `c111-X-1.private.us-east.containers.cloud.ibm.com` hostnames (which are the ones the cluster worker nodes use), where X is in 1,2,3.   Here are those IPs:

```
Brads-MacBook:~ behle$ dig +short c111-1-1.private.us-east.containers.cloud.ibm.com
prod-wdc04-tugboat11-275898.us-east.serviceendpoint.cloud.ibm.com.
166.9.68.134
Brads-MacBook:~ behle$ dig +short c111-2-1.private.us-east.containers.cloud.ibm.com
prod-wdc06-tugboat11-327821.us-east.serviceendpoint.cloud.ibm.com.
166.9.22.215
Brads-MacBook:~ behle$ dig +short c111-3-1.private.us-east.containers.cloud.ibm.com
prod-wdc07-tugboat11-394214.us-east.serviceendpoint.cloud.ibm.com.
166.9.24.198
Brads-MacBook:~ behle$ 
Brads-MacBook:~ behle$ 
Brads-MacBook:~ behle$ 
Brads-MacBook:~ behle$ 
Brads-MacBook:~ behle$ dig +short c111.private.us-east.containers.cloud.ibm.com
prod-us-east-tugboat-487225.us-east.serviceendpoint.cloud.ibm.com.
166.9.22.216
166.9.68.135
166.9.24.199
Brads-MacBook:~ behle$ 
```

And I don't think we we use the IPs from `c111.private.us-east.containers.cloud.ibm.com`

Reply:
Also, don't we also need the public IPs for this new c111 in us-east?

Reply:
closing this pr and created  -https://github.com/IBM-Cloud/kube-samples/pull/192